.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: unit-tests
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node: [10, 12, 14]
+
+    name: ${{ matrix.os }} Node ${{ matrix.node }} cache
+
+    env:
+      OS: ${{ matrix.os }}
+      NODE: ${{ matrix.node }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Get npm cache directory
+        id: npm-cache
+        run: |
+          echo "::set-output name=dir::$(npm config get cache)"
```

Comment:
Shouldn't that use the `yarn cache dir`?

Reply:
Also true.

I'll update the PR.